From 6cad70f25460f3ede0429ce11d5e60946acf1174 Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Fri, 31 Mar 2017 11:23:50 -0400
Subject: [PATCH] Issue 49205 - Fix logconv.pl man page

Description:  Fixed mistake in wording in the logconv.pl man page.  Also
              updated all the man pages with the new pagure ticket url.

https://pagure.io/389-ds-base/issue/49205

Reviewed by: nhosoi(Thanks!)
---
 man/man1/cl-dump.1                  |  6 +++---
 man/man1/dbgen.pl.1                 |  6 +++---
 man/man1/dbscan.1                   |  6 +++---
 man/man1/ds-logpipe.py.1            |  6 +++---
 man/man1/dsktune.1                  |  6 +++---
 man/man1/infadd.1                   |  6 +++---
 man/man1/ldap-agent.1               |  6 +++---
 man/man1/ldclt.1                    |  6 +++---
 man/man1/ldif.1                     |  6 +++---
 man/man1/logconv.pl.1               | 10 +++++-----
 man/man1/migratecred.1              |  6 +++---
 man/man1/mmldif.1                   |  6 +++---
 man/man1/pwdhash.1                  |  6 +++---
 man/man1/readnsstate.1              |  6 +++---
 man/man1/repl-monitor.1             |  8 ++++----
 man/man1/rsearch.1                  |  6 +++---
 man/man8/bak2db.8                   |  6 +++---
 man/man8/bak2db.pl.8                |  6 +++---
 man/man8/cleanallruv.pl.8           |  6 +++---
 man/man8/db2bak.8                   |  6 +++---
 man/man8/db2bak.pl.8                |  6 +++---
 man/man8/db2index.8                 |  6 +++---
 man/man8/db2index.pl.8              |  6 +++---
 man/man8/db2ldif.8                  |  6 +++---
 man/man8/db2ldif.pl.8               |  6 +++---
 man/man8/dbverify.8                 |  6 +++---
 man/man8/dn2rdn.8                   |  6 +++---
 man/man8/fixup-linkedattrs.pl.8     |  6 +++---
 man/man8/fixup-memberof.pl.8        |  6 +++---
 man/man8/ldif2db.8                  |  6 +++---
 man/man8/ldif2db.pl.8               |  6 +++---
 man/man8/ldif2ldap.8                |  6 +++---
 man/man8/migrate-ds.pl.8            | 18 +++++++++---------
 man/man8/monitor.8                  |  6 +++---
 man/man8/ns-accountstatus.pl.8      |  6 +++---
 man/man8/ns-activate.pl.8           |  6 +++---
 man/man8/ns-inactivate.pl.8         |  6 +++---
 man/man8/ns-newpwpolicy.pl.8        |  6 +++---
 man/man8/ns-slapd.8                 |  6 +++---
 man/man8/remove-ds.pl.8             |  6 +++---
 man/man8/restart-dirsrv.8           |  6 +++---
 man/man8/restoreconfig.8            |  6 +++---
 man/man8/saveconfig.8               |  6 +++---
 man/man8/schema-reload.pl.8         |  6 +++---
 man/man8/setup-ds.pl.8              |  6 +++---
 man/man8/start-dirsrv.8             |  6 +++---
 man/man8/status-dirsrv.8            |  6 +++---
 man/man8/stop-dirsrv.8              |  6 +++---
 man/man8/suffix2instance.8          |  6 +++---
 man/man8/syntax-validate.pl.8       |  6 +++---
 man/man8/upgradedb.8                |  6 +++---
 man/man8/upgradednformat.8          |  6 +++---
 man/man8/usn-tombstone-cleanup.pl.8 |  6 +++---
 man/man8/verify-db.pl.8             |  6 +++---
 man/man8/vlvindex.8                 |  6 +++---
 55 files changed, 174 insertions(+), 174 deletions(-)

diff --git a/man/man1/cl-dump.1 b/man/man1/cl-dump.1
index f9dedbe..db736ac 100644
--- a/man/man1/cl-dump.1
+++ b/man/man1/cl-dump.1
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH CL-DUMP 1 "May 18, 2008"
+.TH CL-DUMP 1 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -81,11 +81,11 @@ is running, and from where the server's changelog directory is accessible.
 .SH AUTHOR
 cl-dump was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
 Copyright \(co 2001 Sun Microsystems, Inc. Used by permission.
 .br
-Copyright \(co 2008 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by Michele Baldessari <michele@pupazzo.org>,
 for the Debian project (but may be used by others).
diff --git a/man/man1/dbgen.pl.1 b/man/man1/dbgen.pl.1
index 6f25080..c238c9c 100644
--- a/man/man1/dbgen.pl.1
+++ b/man/man1/dbgen.pl.1
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH DBGEN.PL 1 "May 18, 2008"
+.TH DBGEN.PL 1 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -78,11 +78,11 @@ Add groups containing uniquemembers; generate a group for every 100 user entries
 .SH AUTHOR
 dbgen.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
 Copyright \(co 2001 Sun Microsystems, Inc. Used by permission.
 .br
-Copyright \(co 2008 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by Michele Baldessari <michele@pupazzo.org>,
 for the Debian project (but may be used by others).
diff --git a/man/man1/dbscan.1 b/man/man1/dbscan.1
index e5ff0a7..641a3af 100644
--- a/man/man1/dbscan.1
+++ b/man/man1/dbscan.1
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH DBSCAN 1 "May 18, 2008"
+.TH DBSCAN 1 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -91,11 +91,11 @@ dbscan \fB\-f\fR objectclass.db4
 .SH AUTHOR
 dbscan was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
 Copyright \(co 2001 Sun Microsystems, Inc. Used by permission.
 .br
-Copyright \(co 2008 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by Michele Baldessari <michele@pupazzo.org>,
 for the Debian project (but may be used by others).
diff --git a/man/man1/ds-logpipe.py.1 b/man/man1/ds-logpipe.py.1
index c7ea93d..0a6c15a 100644
--- a/man/man1/ds-logpipe.py.1
+++ b/man/man1/ds-logpipe.py.1
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH DS-LOGPIPE.PY 1 "November 24, 2009"
+.TH DS-LOGPIPE.PY 1 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -90,9 +90,9 @@ The error log and audit log have similarly named configuration attributes e.g. n
 .SH AUTHOR
 ds-logpipe.py was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2009 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This is free software.  You may redistribute copies of it under the terms of
 the Directory Server license found in the LICENSE file of this
diff --git a/man/man1/dsktune.1 b/man/man1/dsktune.1
index 19a6229..b3a5f7b 100644
--- a/man/man1/dsktune.1
+++ b/man/man1/dsktune.1
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH DSKTUNE 1 "May 18, 2008"
+.TH DSKTUNE 1 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -49,11 +49,11 @@ specify alternate server installation directory
 .SH AUTHOR
 dsktune was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
 Copyright \(co 2001 Sun Microsystems, Inc. Used by permission.
 .br
-Copyright \(co 2008 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by Michele Baldessari <michele@pupazzo.org>,
 for the Debian project (but may be used by others).
diff --git a/man/man1/infadd.1 b/man/man1/infadd.1
index 0969019..75c2f42 100644
--- a/man/man1/infadd.1
+++ b/man/man1/infadd.1
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH INFADD 1 "May 18, 2008"
+.TH INFADD 1 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -67,11 +67,11 @@ add binary blob of average size of <size> bytes
 .SH AUTHOR
 infadd was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
 Copyright \(co 2001 Sun Microsystems, Inc. Used by permission.
 .br
-Copyright \(co 2008 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by Michele Baldessari <michele@pupazzo.org>,
 for the Debian project (but may be used by others).
diff --git a/man/man1/ldap-agent.1 b/man/man1/ldap-agent.1
index 50d80c3..f2d6576 100644
--- a/man/man1/ldap-agent.1
+++ b/man/man1/ldap-agent.1
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH LDAP-AGENT 1 "May 18, 2008"
+.TH LDAP-AGENT 1 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -44,11 +44,11 @@ Sample usage:
 .SH AUTHOR
 ldap\-agent was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
 Copyright \(co 2001 Sun Microsystems, Inc. Used by permission.
 .br
-Copyright \(co 2008 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by Michele Baldessari <michele@pupazzo.org>,
 for the Debian project (but may be used by others).
diff --git a/man/man1/ldclt.1 b/man/man1/ldclt.1
index 4105f42..451a88b 100644
--- a/man/man1/ldclt.1
+++ b/man/man1/ldclt.1
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH LDCLT 1 "May 18, 2008"
+.TH LDCLT 1 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -220,11 +220,11 @@ Execution parameters:
 .SH AUTHOR
 ldclt was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
 Copyright \(co 2001 Sun Microsystems, Inc. Used by permission.
 .br
-Copyright \(co 2008 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by Michele Baldessari <michele@pupazzo.org>,
 for the Debian project (but may be used by others).
diff --git a/man/man1/ldif.1 b/man/man1/ldif.1
index f5fa99b..c2398b4 100644
--- a/man/man1/ldif.1
+++ b/man/man1/ldif.1
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH LDIF 1 "May 18, 2008"
+.TH LDIF 1 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -40,11 +40,11 @@ ldif dn < /tmp/ldif
 .SH AUTHOR
 ldif was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
 Copyright \(co 2001 Sun Microsystems, Inc. Used by permission.
 .br
-Copyright \(co 2008 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by Michele Baldessari <michele@pupazzo.org>,
 for the Debian project (but may be used by others).
diff --git a/man/man1/logconv.pl.1 b/man/man1/logconv.pl.1
index e4c1d13..13db49f 100644
--- a/man/man1/logconv.pl.1
+++ b/man/man1/logconv.pl.1
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH LOGCONV.PL 1 "May 18, 2008"
+.TH LOGCONV.PL 1 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -38,7 +38,7 @@ DEFAULT \-> cn=directory manager
 .B \fB\-D, \-\-data\fR <Location for temporary data files>
 DEFAULT \-> /tmp
 .br
-TIP \-> If there is not enough RAM, a RAM disk can be used instead:
+TIP \-> If there is enough RAM, a RAM disk can be used instead:
 
        mkdir /dev/shm/logconv, and use this directory for the "\-D" value.
 .TP
@@ -115,7 +115,7 @@ Examples:
 .IP
 logconv.pl \fB\-s\fR 10 \fB\-V\fR access
 .IP
-logconv.pl \fB\-d\fR "cn=directory manager" /export/server4/slapd\-host/logs/access*
+logconv.pl \fB\-d\fR "cn=directory manager" /var/log/dirsrv/slapd\-host/access*
 .IP
 logconv.pl \fB\-s\fR 50 \fB\-ibgju\fR access*
 .IP
@@ -124,11 +124,11 @@ logconv.pl \fB\-S\fR "[28/Mar/2002:13:14:22 \fB\-0800]\fR" \fB\-E\fR "[28/Mar/20
 .SH AUTHOR
 logconv.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
 Copyright \(co 2001 Sun Microsystems, Inc. Used by permission.
 .br
-Copyright \(co 2008 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by Michele Baldessari <michele@pupazzo.org>,
 for the Debian project (but may be used by others).
diff --git a/man/man1/migratecred.1 b/man/man1/migratecred.1
index 50fbe0a..e935b5f 100644
--- a/man/man1/migratecred.1
+++ b/man/man1/migratecred.1
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH MIGRATECRED 1 "May 18, 2008"
+.TH MIGRATECRED 1 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -50,11 +50,11 @@ New plugin path (of the new instance)
 .SH AUTHOR
 migratecred was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
 Copyright \(co 2001 Sun Microsystems, Inc. Used by permission.
 .br
-Copyright \(co 2008 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by Michele Baldessari <michele@pupazzo.org>,
 for the Debian project (but may be used by others).
diff --git a/man/man1/mmldif.1 b/man/man1/mmldif.1
index e3f31ca..bde95d1 100644
--- a/man/man1/mmldif.1
+++ b/man/man1/mmldif.1
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH MMLDIF 1 "May 18, 2008"
+.TH MMLDIF 1 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -46,11 +46,11 @@ Write authoritative data to this file
 .SH AUTHOR
 mmldif was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
 Copyright \(co 2001 Sun Microsystems, Inc. Used by permission.
 .br
-Copyright \(co 2008 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by Michele Baldessari <michele@pupazzo.org>,
 for the Debian project (but may be used by others).
diff --git a/man/man1/pwdhash.1 b/man/man1/pwdhash.1
index 3ef5c9d..96194b3 100644
--- a/man/man1/pwdhash.1
+++ b/man/man1/pwdhash.1
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH PWDHASH 1 "May 18, 2008"
+.TH PWDHASH 1 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -47,11 +47,11 @@ Takes the password schema directly from the ns-slapd configuration
 .SH AUTHOR
 dbscan was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
 Copyright \(co 2001 Sun Microsystems, Inc. Used by permission.
 .br
-Copyright \(co 2008 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by Michele Baldessari <michele@pupazzo.org>,
 for the Debian project (but may be used by others).
diff --git a/man/man1/readnsstate.1 b/man/man1/readnsstate.1
index 0edf352..b18f49a 100644
--- a/man/man1/readnsstate.1
+++ b/man/man1/readnsstate.1
@@ -1,4 +1,4 @@
-.TH READNSSTATE 1 "May 13 2016"
+.TH READNSSTATE 1 "March 31, 2017"
 .SH NAME
 readnsstate \- interpret the contents of cn=replica's nsState value
 .B readnsstate
@@ -38,9 +38,9 @@ For replica cn=replica,cn=dc\3Dexample\2Cdc\3Dcom,cn=mapping tree,cn=config
 .SH AUTHOR
 readnsstate was written by the 389 Project by richm.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2016 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by William Brown <wibrown@redhat.com>,
 for the 389 Directory Server Project.
diff --git a/man/man1/repl-monitor.1 b/man/man1/repl-monitor.1
index 17b9c4b..2e4fc77 100644
--- a/man/man1/repl-monitor.1
+++ b/man/man1/repl-monitor.1
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH REPL-MONITOR 1 "Jun 28, 2016"
+.TH REPL-MONITOR 1 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -121,16 +121,16 @@ MY_SYSTEM2 = localhost2.localdomain:3892
 .SH AUTHOR
 repl-monitor was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
 Copyright \(co 2001 Sun Microsystems, Inc. Used by permission.
 .br
-Copyright \(co 2016 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by Michele Baldessari <michele@pupazzo.org>,
 for the Debian project (but may be used by others).
 .br
-Manual page updated by Mark Reynolds <mreynolds@redhat.com> 6/28/2016
+Manual page updated by Mark Reynolds <mreynolds@redhat.com> 3/31/2017
 .br
 This is free software.  You may redistribute copies of it under the terms of
 the Directory Server license found in the LICENSE file of this
diff --git a/man/man1/rsearch.1 b/man/man1/rsearch.1
index 319bfc1..ec269a5 100644
--- a/man/man1/rsearch.1
+++ b/man/man1/rsearch.1
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH RSEARCH 1 "May 18, 2008"
+.TH RSEARCH 1 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -123,11 +123,11 @@ Use \fB\-B\fR file for binding; ignored if \fB\-B\fR is not given
 .SH AUTHOR
 rsearch was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
 Copyright \(co 2001 Sun Microsystems, Inc. Used by permission.
 .br
-Copyright \(co 2008 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by Michele Baldessari <michele@pupazzo.org>,
 for the Debian project (but may be used by others).
diff --git a/man/man8/bak2db.8 b/man/man8/bak2db.8
index 77864728..74e5e58 100644
--- a/man/man8/bak2db.8
+++ b/man/man8/bak2db.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH BAK2DB 8 "Mar 5, 2013"
+.TH BAK2DB 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -57,6 +57,6 @@ to standard error.
 .SH AUTHOR
 bak2db was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/bak2db.pl.8 b/man/man8/bak2db.pl.8
index 1bb76c5..01a41c1 100644
--- a/man/man8/bak2db.pl.8
+++ b/man/man8/bak2db.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH BAK2DB.PL 8 "Mar 5, 2013"
+.TH BAK2DB.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -73,6 +73,6 @@ to standard error.
 .SH AUTHOR
 bak2db.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/cleanallruv.pl.8 b/man/man8/cleanallruv.pl.8
index 55678ac..3afc688 100644
--- a/man/man8/cleanallruv.pl.8
+++ b/man/man8/cleanallruv.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH CLEANALLRUV.PL 8 " Mar 5, 2013"
+.TH CLEANALLRUV.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -76,6 +76,6 @@ to standard error.
 .SH AUTHOR
 cleanallruv.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/db2bak.8 b/man/man8/db2bak.8
index 5de017e..727867a 100644
--- a/man/man8/db2bak.8
+++ b/man/man8/db2bak.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH DB2BAK 8 "Mar 5, 2013"
+.TH DB2BAK 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -58,6 +58,6 @@ to standard error.
 .SH AUTHOR
 db2bak was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/db2bak.pl.8 b/man/man8/db2bak.pl.8
index 9a34d51..a752885 100644
--- a/man/man8/db2bak.pl.8
+++ b/man/man8/db2bak.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH DB2BAK.PL 8 "Mar 5, 2013"
+.TH DB2BAK.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -72,6 +72,6 @@ to standard error.
 .SH AUTHOR
 db2bak.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/db2index.8 b/man/man8/db2index.8
index 1e70cc9..f70ba67 100644
--- a/man/man8/db2index.8
+++ b/man/man8/db2index.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH DB2INDEX 8 "Mar 5, 2013"
+.TH DB2INDEX 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -62,6 +62,6 @@ to standard error.
 .SH AUTHOR
 db2index was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/db2index.pl.8 b/man/man8/db2index.pl.8
index 4ff9c7a..7297fae 100644
--- a/man/man8/db2index.pl.8
+++ b/man/man8/db2index.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH DB2INDEX.PL 8 "Mar 5, 2013"
+.TH DB2INDEX.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -76,6 +76,6 @@ to standard error.
 .SH AUTHOR
 db2index.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/db2ldif.8 b/man/man8/db2ldif.8
index 2a787f2..31f686b 100644
--- a/man/man8/db2ldif.8
+++ b/man/man8/db2ldif.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH DB2LDIF 8 "Mar 5, 2013"
+.TH DB2LDIF 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -101,6 +101,6 @@ to standard error.
 .SH AUTHOR
 db2ldif was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/db2ldif.pl.8 b/man/man8/db2ldif.pl.8
index f02d3ed..babd225 100644
--- a/man/man8/db2ldif.pl.8
+++ b/man/man8/db2ldif.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH DB2LDIF.PL 8 "Mar 5, 2013"
+.TH DB2LDIF.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -115,6 +115,6 @@ to standard error.
 .SH AUTHOR
 db2ldif.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/dbverify.8 b/man/man8/dbverify.8
index c74747a..256e0aa 100644
--- a/man/man8/dbverify.8
+++ b/man/man8/dbverify.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH DBVERIFY 8 "Mar 5, 2013"
+.TH DBVERIFY 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -58,6 +58,6 @@ to standard error.
 .SH AUTHOR
 dbverify was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/dn2rdn.8 b/man/man8/dn2rdn.8
index d6cd3cc..98d74da 100644
--- a/man/man8/dn2rdn.8
+++ b/man/man8/dn2rdn.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH DN2RDN 8 "Mar 5, 2013"
+.TH DN2RDN 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -53,6 +53,6 @@ to standard error.
 .SH AUTHOR
 dn2rdn was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/fixup-linkedattrs.pl.8 b/man/man8/fixup-linkedattrs.pl.8
index ee484c8..1189cb8 100644
--- a/man/man8/fixup-linkedattrs.pl.8
+++ b/man/man8/fixup-linkedattrs.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH FIXUP-LINKEDATTRS.PL 8 "Mar 5, 2013"
+.TH FIXUP-LINKEDATTRS.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -67,6 +67,6 @@ to standard error.
 .SH AUTHOR
 fixup-linkedattrs.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/fixup-memberof.pl.8 b/man/man8/fixup-memberof.pl.8
index 55b7503..f80043c 100644
--- a/man/man8/fixup-memberof.pl.8
+++ b/man/man8/fixup-memberof.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH FIXUP-MEMBEROF.PL 8 "Mar 5, 2013"
+.TH FIXUP-MEMBEROF.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -71,6 +71,6 @@ to standard error.
 .SH AUTHOR
 fixup-memberof.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/ldif2db.8 b/man/man8/ldif2db.8
index a5db3ea..9e3b149 100644
--- a/man/man8/ldif2db.8
+++ b/man/man8/ldif2db.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH LDIF2DB 8 "Mar 5, 2013"
+.TH LDIF2DB 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -84,6 +84,6 @@ to standard error.
 .SH AUTHOR
 ldif2db was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/ldif2db.pl.8 b/man/man8/ldif2db.pl.8
index cc3e316..3c02d03 100644
--- a/man/man8/ldif2db.pl.8
+++ b/man/man8/ldif2db.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH LDIF2DB.PL 8 "Mar 5, 2013"
+.TH LDIF2DB.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -93,6 +93,6 @@ to standard error.
 .SH AUTHOR
 ldif2db.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/ldif2ldap.8 b/man/man8/ldif2ldap.8
index 117a1f7..b89f464 100644
--- a/man/man8/ldif2ldap.8
+++ b/man/man8/ldif2ldap.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH LDIF2LDAP 8 "Mar 5, 2013"
+.TH LDIF2LDAP 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -57,6 +57,6 @@ to standard error.
 .SH AUTHOR
 ldif2ldap was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/migrate-ds.pl.8 b/man/man8/migrate-ds.pl.8
index 65a62d0..24eac5a 100644
--- a/man/man8/migrate-ds.pl.8
+++ b/man/man8/migrate-ds.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH MIGRATE-DS.PL 8 "May 18, 2008"
+.TH MIGRATE-DS.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -121,15 +121,15 @@ for migration.  Changelog information will not be migrated, and replicas
 will need to be reinitialized (if migrating masters or hubs).  This type
 of migration requires that all of your old databases have been dumped
 to LDIF format, and the LDIF file must be in the default database directory
-(usually /opt/fedora\-ds/slapd\-instance/db), and the LDIF file must have
+(usually /var/lib/dirsrv/slapd\-instance/db), and the LDIF file must have
 the same name as the database instance directory, with a ".ldif".  For
 example, if you have
 .IP
 .ad l
 .nf
-/opt/fedora\-ds/slapd\-instance/db/userRoot/
+/var/lib/dirsrv/slapd\-instance/db/userRoot/
 and
-/opt/fedora\-ds/slapd\-instance/db/NetscapeRoot/
+/var/lib/dirsrv/slapd\-instance/db/NetscapeRoot/
 .na
 .fi
 .PP
@@ -137,9 +137,9 @@ you must first use db2ldif to export these databases to LDIF e.g.
 .IP
 .ad l
 .nf
-cd /opt/fedora\-ds/slapd\-instance
-\&./db2ldif \fB\-n\fR userRoot \fB\-a\fR /opt/fedora\-ds/slapd\-instance/db/userRoot.ldif and
-\&./db2ldif \fB\-n\fR NetscapeRoot \fB\-a\fR /opt/fedora\-ds/slapd\-instance/db/NetscapeRoot.ldif
+cd /var/lib/dirsrv/slapd\-instance
+\&./db2ldif \fB\-n\fR userRoot \fB\-a\fR /var/lib/dirsrv/slapd\-instance/ldif/userRoot.ldif and
+\&./db2ldif \fB\-n\fR NetscapeRoot \fB\-a\fR /var/lib/dirsrv/slapd\-instance/ldif/NetscapeRoot.ldif
 .fi
 .na
 .PP
@@ -151,9 +151,9 @@ directory on the destination machine.
 .SH AUTHOR
 migrate-ds.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2008 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by Michele Baldessari <michele@pupazzo.org>,
 for the Debian project (but may be used by others).
diff --git a/man/man8/monitor.8 b/man/man8/monitor.8
index 1cc2166..1e75fcf 100644
--- a/man/man8/monitor.8
+++ b/man/man8/monitor.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH MONITOR 8 "Mar 5, 2013"
+.TH MONITOR 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -62,6 +62,6 @@ to standard error.
 .SH AUTHOR
 monitor was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/ns-accountstatus.pl.8 b/man/man8/ns-accountstatus.pl.8
index 9ffc4d3..ceba3e6 100644
--- a/man/man8/ns-accountstatus.pl.8
+++ b/man/man8/ns-accountstatus.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH NS-ACCOUNTSTATUS.PL 8 "Feb 8, 2016"
+.TH NS-ACCOUNTSTATUS.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -90,6 +90,6 @@ to standard error.
 .SH AUTHOR
 ns-accountstatus.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2016 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/ns-activate.pl.8 b/man/man8/ns-activate.pl.8
index 4b6f46d..2d4b82c 100644
--- a/man/man8/ns-activate.pl.8
+++ b/man/man8/ns-activate.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH NS-ACTIVATE.PL 8 "Mar 5, 2013"
+.TH NS-ACTIVATE.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -68,6 +68,6 @@ to standard error.
 .SH AUTHOR
 ns-activate.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/ns-inactivate.pl.8 b/man/man8/ns-inactivate.pl.8
index 9e8ad77..f71a63f 100644
--- a/man/man8/ns-inactivate.pl.8
+++ b/man/man8/ns-inactivate.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH NS-INACTIVATE.PL 8 "Mar 5, 2013"
+.TH NS-INACTIVATE.PL 8 "Mar 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -67,6 +67,6 @@ to standard error.
 .SH AUTHOR
 ns-inactivate.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/ns-newpwpolicy.pl.8 b/man/man8/ns-newpwpolicy.pl.8
index 1c38748..795bdc9 100644
--- a/man/man8/ns-newpwpolicy.pl.8
+++ b/man/man8/ns-newpwpolicy.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH NS-NEWPWPOLICY.PL 8 "Mar 5, 2013"
+.TH NS-NEWPWPOLICY.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -74,6 +74,6 @@ to standard error.
 .SH AUTHOR
 ns-newpwpolicy.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/ns-slapd.8 b/man/man8/ns-slapd.8
index 7c61533..96f995e 100644
--- a/man/man8/ns-slapd.8
+++ b/man/man8/ns-slapd.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH NS-SLAPD 8 "May 18, 2008"
+.TH NS-SLAPD 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -45,11 +45,11 @@ Specifies file where the pid of the process will be stored
 .SH AUTHOR
 ns-slapd was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
 Copyright \(co 2001 Sun Microsystems, Inc. Used by permission.
 .br
-Copyright \(co 2008 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by Michele Baldessari <michele@pupazzo.org>,
 for the Debian project (but may be used by others).
diff --git a/man/man8/remove-ds.pl.8 b/man/man8/remove-ds.pl.8
index 0568ff8..5d60e47 100644
--- a/man/man8/remove-ds.pl.8
+++ b/man/man8/remove-ds.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH REMOVE-DS.PL 8 "Feb 13, 2009"
+.TH REMOVE-DS.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -48,9 +48,9 @@ The full name of the instance to remove (e.g. slapd\(hyexample)
 .SH AUTHOR
 remove-ds.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2009 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This is free software.  You may redistribute copies of it under the terms of
 the Directory Server license found in the LICENSE file of this
diff --git a/man/man8/restart-dirsrv.8 b/man/man8/restart-dirsrv.8
index c8af2ce..c82ec2b 100644
--- a/man/man8/restart-dirsrv.8
+++ b/man/man8/restart-dirsrv.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH RESTART-DIRSRV 8 "Jun 8, 2010"
+.TH RESTART-DIRSRV 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -40,9 +40,9 @@ Sample usage:
 .SH AUTHOR
 restart\-dirsrv was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2010 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This is free software.  You may redistribute copies of it under the terms of
 the Directory Server license found in the LICENSE file of this
diff --git a/man/man8/restoreconfig.8 b/man/man8/restoreconfig.8
index 18b27f5..97642b6 100644
--- a/man/man8/restoreconfig.8
+++ b/man/man8/restoreconfig.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH RESTORECONFIG 8 "Mar 5, 2013"
+.TH RESTORECONFIG 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -43,6 +43,6 @@ to standard error.
 .SH AUTHOR
 restoreconfig was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/saveconfig.8 b/man/man8/saveconfig.8
index 8ba8a1d..1597043 100644
--- a/man/man8/saveconfig.8
+++ b/man/man8/saveconfig.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH SAVECONFIG 8 "Mar 5, 2013"
+.TH SAVECONFIG 8 "March 31, 2013"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -43,6 +43,6 @@ to standard error.
 .SH AUTHOR
 saveconfig was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/schema-reload.pl.8 b/man/man8/schema-reload.pl.8
index 17380cf..85797f3 100644
--- a/man/man8/schema-reload.pl.8
+++ b/man/man8/schema-reload.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH SCHEMA-RELOAD.PL 8 "Mar 5, 2013"
+.TH SCHEMA-RELOAD.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -65,6 +65,6 @@ to standard error.
 .SH AUTHOR
 schema-reload.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/setup-ds.pl.8 b/man/man8/setup-ds.pl.8
index 11e0966..b491e18 100644
--- a/man/man8/setup-ds.pl.8
+++ b/man/man8/setup-ds.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH SETUP-DS.PL 8 "May 18, 2008"
+.TH SETUP-DS.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -103,9 +103,9 @@ Offline \- servers \fBmust be shutdown\fR - no username or password required
 .SH AUTHOR
 setup-ds.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2008 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This manual page was written by Michele Baldessari <michele@pupazzo.org>,
 for the Debian project (but may be used by others).
diff --git a/man/man8/start-dirsrv.8 b/man/man8/start-dirsrv.8
index f34da66..34f3988 100644
--- a/man/man8/start-dirsrv.8
+++ b/man/man8/start-dirsrv.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH START-DIRSRV 8 "Jun 8, 2010"
+.TH START-DIRSRV 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -40,9 +40,9 @@ Sample usage:
 .SH AUTHOR
 start\-dirsrv was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2010 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This is free software.  You may redistribute copies of it under the terms of
 the Directory Server license found in the LICENSE file of this
diff --git a/man/man8/status-dirsrv.8 b/man/man8/status-dirsrv.8
index 83844b3..862c871 100644
--- a/man/man8/status-dirsrv.8
+++ b/man/man8/status-dirsrv.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH STATUS-DIRSRV 8 "Jan 20, 2016"
+.TH STATUS-DIRSRV 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -45,9 +45,9 @@ If a single Directory Server instance is specified, 0 is returned if the instanc
 .SH AUTHOR
 status\-dirsrv was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2016 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This is free software.  You may redistribute copies of it under the terms of
 the Directory Server license found in the LICENSE file of this
diff --git a/man/man8/stop-dirsrv.8 b/man/man8/stop-dirsrv.8
index 08bf493..fdca0a9 100644
--- a/man/man8/stop-dirsrv.8
+++ b/man/man8/stop-dirsrv.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH STOP-DIRSRV 8 "Jun 8, 2010"
+.TH STOP-DIRSRV 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -40,9 +40,9 @@ Sample usage:
 .SH AUTHOR
 stop\-dirsrv was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2010 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 .br
 This is free software.  You may redistribute copies of it under the terms of
 the Directory Server license found in the LICENSE file of this
diff --git a/man/man8/suffix2instance.8 b/man/man8/suffix2instance.8
index 8e30f4a..0e4cf69 100644
--- a/man/man8/suffix2instance.8
+++ b/man/man8/suffix2instance.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH SUFFIX2INSTANCE 8 "Mar 5, 2013"
+.TH SUFFIX2INSTANCE 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -46,6 +46,6 @@ to standard error.
 .SH AUTHOR
 suffix2instance was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/syntax-validate.pl.8 b/man/man8/syntax-validate.pl.8
index ece2d59..d34c49d 100644
--- a/man/man8/syntax-validate.pl.8
+++ b/man/man8/syntax-validate.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH SYNTAX-VALIDATE.PL 8 "Mar 5, 2013"
+.TH SYNTAX-VALIDATE.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -68,6 +68,6 @@ to standard error.
 .SH AUTHOR
 syntax-validate.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/upgradedb.8 b/man/man8/upgradedb.8
index 495c722..70d1c5f 100644
--- a/man/man8/upgradedb.8
+++ b/man/man8/upgradedb.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH UPGRADEDB 8 "Mar 5, 2013"
+.TH UPGRADEDB 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -50,7 +50,7 @@ to standard error.
 .SH AUTHOR
 upgradedb was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
 
diff --git a/man/man8/upgradednformat.8 b/man/man8/upgradednformat.8
index 3f3f58f..7513994 100644
--- a/man/man8/upgradednformat.8
+++ b/man/man8/upgradednformat.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH UPGRADEDNFORMAT 8 "Mar 5, 2013"
+.TH UPGRADEDNFORMAT 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -50,6 +50,6 @@ to standard error.
 .SH AUTHOR
 upgradednformat was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/usn-tombstone-cleanup.pl.8 b/man/man8/usn-tombstone-cleanup.pl.8
index f78b230..c5aabc5 100644
--- a/man/man8/usn-tombstone-cleanup.pl.8
+++ b/man/man8/usn-tombstone-cleanup.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH USN-TOMBSTONE-CLEANUP.PL 8 "Mar 5, 2013"
+.TH USN-TOMBSTONE-CLEANUP.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -71,6 +71,6 @@ to standard error.
 .SH AUTHOR
 usn-tombstone-cleanup.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/verify-db.pl.8 b/man/man8/verify-db.pl.8
index 500b713..0de3af5 100644
--- a/man/man8/verify-db.pl.8
+++ b/man/man8/verify-db.pl.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH VERIFY-DB.PL 8 "Mar 5, 2013"
+.TH VERIFY-DB.PL 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -44,6 +44,6 @@ to standard error.
 .SH AUTHOR
 verify-db.pl was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
diff --git a/man/man8/vlvindex.8 b/man/man8/vlvindex.8
index 154f1c2..f3e1748 100644
--- a/man/man8/vlvindex.8
+++ b/man/man8/vlvindex.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH VLVINDEX 8 "Mar 5, 2013"
+.TH VLVINDEX 8 "March 31, 2017"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .\" Some roff macros, for reference:
@@ -53,6 +53,6 @@ to standard error.
 .SH AUTHOR
 vlvindex was written by the 389 Project.
 .SH "REPORTING BUGS"
-Report bugs to https://fedorahosted.org/389/newticket.
+Report bugs to https://pagure.io/389-ds-base/new_issue
 .SH COPYRIGHT
-Copyright \(co 2013 Red Hat, Inc.
+Copyright \(co 2017 Red Hat, Inc.
-- 
2.9.3

